v3-nginx/Dockerfile

```
@@ -61,7 +61,8 @@ RUN set -eux; \
     ./build.sh; \
     git submodule init; \
     git submodule update; \
-    ./configure --with-yajl=/sources/yajl/build/yajl-${YAJL_VERSION}/ --with-geoip --with-pcre2="$(pcre2-config --prefix)"; \
+    sed -i -e 's/^PCRE2_POSSIBLE_LIB_NAMES=\(.*\)"/PCRE2_POSSIBLE_LIB_NAMES=\1 libpcre2-8"/' build/pcre2.m4 ; \
```

Not sure this works: use `$(pcre2-config --prefix)` in both instances, instead of hard coding `libpcre2-8`.
```suggestion
    # patch make file to support Debian package names (see https://github.com/SpiderLabs/ModSecurity/issues/2810) \
    sed -i -e "s/^PCRE2_POSSIBLE_LIB_NAMES=\(.*\)\"/PCRE2_POSSIBLE_LIB_NAMES=\1 $(pcre2-config --prefix)\""/ build/pcre2.m4; \
```

---

Definitely `$(pre2-config --prefix)` doesn't do what we want. You want the name of the package. And it is specific to debian and alpine. Both name it the same way, `libpcre2-8`. So... the working way was the one before the suggestion, probably. Reverting suggestion.

---

Oh, I see. `$(pcre2-config --prefix)` returns the installation directory prefix, not the package name. Makes sense.